packages/grafana-ui/src/components/OptionsUI/mappings.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 
-import { FieldConfigEditorProps, ValueMapping, ValueMappingFieldConfigSettings } from '@grafana/data';
-import { ValueMappingsEditor } from '../ValueMappingsEditor/ValueMappingsEditor';
+import {
+  deprecationWarning,
+  FieldConfigEditorProps,
+  ValueMapping,
+  ValueMappingFieldConfigSettings,
+} from '@grafana/data';
+import { ValueMappingsEditor } from '../../../../../public/app/features/dimensions/editors/ValueMappingsEditor/ValueMappingsEditor';
```

Comment:
an import from grafana ui to the app is not allowed this will likley cause issues when trying to build an publish this as standalone package

Reply:
This is good to know, thanks!